### DISCUSSION TITLE
Single observables

### DISCUSSION THREAD

**Discussion by aerabi** [ReactiveX/rxjs#6397] [Ideas / Feature request]
RxJava has different observable types based on the number of elements they emit. One of them is `Single` which contains only one element.

Having a specific observable type for `Single` can improve readability of the code. Also, one can support specific operators for it that are not available for normal observables.

**Comment by tmair** [comment]
I think this suggestion has already come up here https://github.com/ReactiveX/rxjs/discussions/6340

**Reply by voliva** [reply]
Yep, there's a very interesting talk from @cartant and @kolodny about it from the 2020's london rxjs.live: https://www.youtube.com/watch?v=vC1KXMo97XI
